src/main/java/edu/umich/tl/Course.java

```diff
@@ -36,5 +36,18 @@ public Course setEndDate(DateTime endDate) {
 		this.endDate = endDate;
 		return this;
 	}
+	
+	public static String getCourseHeader() {
+		StringBuilder courseHeader=new StringBuilder();
+		courseHeader.append("COURSE_NAME");courseHeader.append(',');
+		courseHeader.append("COURSE_ID");courseHeader.append('\n');
+		return courseHeader.toString();
+	}
```

Comment:
are you talking about line- 39? I felt if we have a line gap between functions will be good for readability. is this not a programming best practices?